packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -241,6 +243,41 @@ export const buf2hex = (buffer: Buffer): string => {
   return hexParts.join('');
 };
 
+/**
+ * 
+ *  @description Gets Tezos address (PKH) from Public Key
+ * 
+ *  @param publicKey Public Key
+ *  @returns A string of the Tezos address (PKH) that was derived from the given Public Key
+ */
+ export const getPkhfromPk = (publicKey: string): string => {
+   let encodingPrefix;
+   let prefixLen;
+
+  const keyPrefix = validatePkAndExtractPrefix(publicKey);
+  const decoded = b58cdecode(publicKey, prefix[keyPrefix]);
+
+  switch (keyPrefix) {
+    case Prefix.EDPK:
+      encodingPrefix = prefix['tz1'];
```

Comment:
Consider using "prefix[Prefix.TZ1]" instead of "prefix['tz1']"

Reply:
good catch, will address